Resolves final ADV errors for backup and restore MS docs (#106398)
openshift/openshift-docs: modules/microshift-about-data-backups.adoc

//
// * microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc

:_mod-docs-content-type: CONCEPT
[id="microshift-about-data-backups_{context}"]
= About backing up and restoring {microshift-short} data

[role="_abstract"]

> Backing up and restoring {microshift-short} data applies to the database only, and application data is not included. Before you can create a manual backup, greenboot health checks must finish running and you must stop the {microshift-short} service.

Reviewer: ```suggestion
Backing up and restoring {microshift-short} data applies to the database only, and not to any application data. Before you can create a manual backup, greenboot health checks must finish running and you must stop the {microshift-short} service.
```
